lib/experimental/fonts-library/class-wp-font-family-utils.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Fonts Family Utils class.
+ *
+ * This file contains utils fot Font Family class.
+ *
+ * @package    Gutenberg
+ * @subpackage Fonts Library
+ * @since      X.X.X
+ */
+
+/**
+ * Fonts Library class.
+ */
+class WP_Font_Family_Utils {
```

Comment:
If the `@package` is `Gutenberg`, should these classes also be namespaced to `Gutenberg` rather than `WP` until they are merged to core? I'm not so familiar with the process for introducing a new API. 

```suggestion
class Gutenberg_Font_Family_Utils {
```

Reply:
I'm thinking it should use the `WP_` prefix when the class is intended to be merged into WordPress Core. This is consistent with other classes that were or will be merged into Core, such as `WP_Theme_JSON`, `WP_Font_Face`, the Interactivity API, etc.

Reply:
Good question Jeff, I was wondering the same.
Thanks, Tonya for the answer.

Reply:
Good question Jeff, I was wondering the same.
Thanks, Tonya for the answer.